packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1141,6 +1149,16 @@ abstract class _RenderCustomClip<T> extends RenderProxyBox {
   T get _defaultClip;
   T _clip;
 
+  /// {@template flutter.widget.clipper.clipBehavior}
+  /// Controls how to clip when clipper is null. Otherwise, clipper controls the
```

Comment:
all identifiers of members in docs should be in `[]` every time (and all identifiers of arguments in docs should be in \`\` every time, but that doesn't apply here)